CONTRIBUTING.md

```diff
@@ -1,20 +1,147 @@
 
-### Working with issues
+**The team recommends using [VS Code](https://code.visualstudio.com/download) as the preferred text editor for working on code, but feel free to utilize a text editor of your preference.**
 
-- Explain how to contribute to an existing issue.
+**If you have any additional questions about your contribution process, please feel free to reach out to the team in the [#ops](https://hackforla.slack.com/archives/CV7QGL66B) Slack channel.**
+<br><br>
 
-### Working with forks and branches
+## **Table of Contents**
+- [**How to Contribute to DevOps**](#how-to-contribute-to-devops)
+- [**Table of Contents**](#table-of-contents)
+- [**Setting up the local development environment**](#setting-up-the-local-development-environment)
+    - [**Creating a personal AWS account**](#creating-a-personal-aws-account)
+    - [**Installing Terraform**](#installing-terraform)
+    - [**Installing Terraform docs**](#installing-terraform-docs)
+    - [**Fork the repository**](#fork-the-repository)
+    - [**Clone (Create) a copy on your computer**](#clone-create-a-copy-on-your-computer)
+        - [**Verify `origin` remote url**](#verify-origin-remote-url)
+        - [**What if you accidentally cloned using the repository URL from the HackForLA Github (instead of the fork on your Github)?**](#what-if-you-accidentally-cloned-using-the-repository-url-from-the-hackforla-github-instead-of-the-fork-on-your-github)
+            - [**i. Reset `origin` remote url**](#i-reset-origin-remote-url)
+            - [**ii. Add an `upstream` remote**](#ii-add-an-upstream-remote)
 
-- Explain your guidelines here.
+## **Setting up the local development environment**
 
+### **Creating a personal AWS account**
 
-### Working with pull requests and reviews
+- Go to [amazon.com](https://aws.amazon.com/) and click `"Sign In to the Console" > "Create a new AWS account."` 
+- Enter your email, create a password, and input your basic details. Then provide your contact information and complete the identity verification process.
+- Enter your credit/debit card information for billing purposes, opt for the free basic support.
+- Agree to the AWS Customer Agreement and Service Terms, complete the registration by clicking `"Create Account and Continue"`, verify your phone number via text or call, confirm your email address following the instructions in the confirmation email, and finally sign in to access your new AWS account using your email and password.
 
-- Explain your process.
\ No newline at end of file
+<sub>[Back to Table of Contents](#table-of-contents)</sub>
+***
+
+### **Installing Terraform**
+
+We are using the [Official HashiCorp install instructions](https://developer.hashicorp.com/terraform/install) for installing terraform.
```

Comment:
``
```suggestion
Use the [Official HashiCorp install instructions](https://developer.hashicorp.com/terraform/install) for installing terraform.
```
There's nothing really wrong here - just removing pronouns to standardize and simplify